ui/ui-components/components/export/List.tsx

```diff
@@ -77,18 +77,31 @@ export default function ExportsList(props) {
   }
 
   function getErrorRow(_export: Models.ExportType) {
-    if (!_export.errorMessage) {
-      return null;
-    }
+    if (!_export.errorMessage) return null;
+
+    const delayMessage =
+      ["pending", "processing"].includes(_export.state) &&
+      _export.sendAt > new Date().getTime()
```

Comment:
There are some potential shenanigans here about time drift between the server and browser... but I don't think it will be that bad, as this is just for the UI and not processing...